comp/metadata/inventoryagent/inventoryagentimpl/inventoryagent.go

```diff
@@ -297,6 +297,7 @@ func (ia *inventoryagent) fetchSystemProbeMetadata() {
 	ia.data["system_probe_track_udp_4_connections"] = getBoolSysProbe("network_config.collect_udp_v4")
 	ia.data["system_probe_track_udp_6_connections"] = getBoolSysProbe("network_config.collect_udp_v6")
 	ia.data["system_probe_protocol_classification_enabled"] = getBoolSysProbe("network_config.enable_protocol_classification")
+	ia.data["system_probe_ringbuffers_enabled"] = getBoolSysProbe("network_config.enable_ringbuffers")
```

Comment:
imho, `system_probe_ringbuffers_enabled` does not fit here
we should move it to npm specific